### PR TITLE
Chords and Mensuration Sign's Orientation and Loc

### PR DESCRIPTION
**Pull request by martha-thomae**
Add support for translating the orientation of mensuration signs from CMME to MEI. Also add a few comments for readability.

### CODE REVIEW

File: src/iocmme.cpp [hunk 1]
```diff
@@ -423,6 +428,27 @@ void CmmeInput::CreateBarline(pugi::xml_node barlineNode)
     m_currentContainer->AddChild(barLine);
 }
 
+void CmmeInput::CreateChord(pugi::xml_node chordNode)
+{
+    assert(m_currentContainer);
+
+    Chord *chord = new Chord();
```
Comment (lpugin) [inline src/iocmme.cpp:435]: What about the chord duration? I guess in CMME it is given in the notes (each of them). The practice in Verovio is to expect the longest duration to be given in the chord

File: src/iocmme.cpp [hunk 2]
```diff
@@ -574,12 +603,43 @@ void CmmeInput::CreateMensuration(pugi::xml_node mensurationNode)
     else if (signValue == "C") {
         mensur->SetSign(MENSURATIONSIGN_C);
     }
-    else {
+    else if (signValue != "") {
         LogWarning("Unsupported mesuration sign in CMME (not 'O' or 'C')");
     }
-
+    /// Mensuration/Sign/Dot to @dot
```
Comment (lpugin) [inline src/iocmme.cpp:609]: Add empty line for readability

File: src/iocmme.cpp [hunk 3]
```diff
@@ -574,12 +603,43 @@ void CmmeInput::CreateMensuration(pugi::xml_node mensurationNode)
...
     pugi::xml_node dotNode = (signNode) ? signNode.child("Dot") : pugi::xml_node(NULL);
     mensur->SetDot(((dotNode) ? BOOLEAN_true : BOOLEAN_false));
+    /// Sign/Strokes to @slash
+    int strokes = this->ChildAsInt(signNode, "Strokes");
+    if (strokes != VRV_UNSET) {
+        mensur->SetSlash(strokes);
+    }
+    /// Mensuration/Sign/Orientation to @orient
```
Comment (lpugin) [inline src/iocmme.cpp:617]: Add line before

File: src/iocmme.cpp [hunk 4]
```diff
@@ -574,12 +603,43 @@ void CmmeInput::CreateMensuration(pugi::xml_node mensurationNode)
...
+    }
+    else if (orientation == "90CW") {
+        mensur->SetOrient(ORIENTATION_90CW);
+    }
+    else if (orientation == "90CCW") {
+        mensur->SetOrient(ORIENTATION_90CCW);
+    }
+    else if (orientation != "") {
+        LogWarning("Unsupported mesuration orientation in CMME (not 'Reversed' or '90CW' or '90CCW')");
+    }
```
Comment (lpugin) [inline src/iocmme.cpp:630]: Make a map

File: src/iocmme.cpp (hunk 1 above)
Comment (DILewis) [reply src/iocmme.cpp:436]: I *think* my latest commit does this, but please check

**Comment by lpugin** [comment]
Merging now, will be adjusted if needed. Thanks

**Pull request merged**